sources/packages/api/src-sql/EducationProgramsOfferings/Alter-education-program-offerings.sql

```diff
@@ -0,0 +1,73 @@
+-- Add year_of_study to education_programs_offerings table.
+ALTER TABLE
+    sims.education_programs_offerings
+ADD
+    COLUMN IF NOT EXISTS year_of_study INT NOT NULL DEFAULT 1;
+
+COMMENT ON COLUMN sims.education_programs_offerings.year_of_study IS 'Year of study of the study period.';
+
+-- Add show_yr_of_study to education_programs_offerings table.
+ALTER TABLE
+    sims.education_programs_offerings
+ADD
+    COLUMN IF NOT EXISTS show_yr_of_study BOOLEAN;
+
+COMMENT ON COLUMN sims.education_programs_offerings.show_yr_of_study IS 'This value determins if we show year of study to students.';
+
+-- Add has_offering_wil_component to education_programs_offerings table.
```

Comment:
what is `wil` component? could you explain

Reply:
WIL Abbreviated in comment.

Reply:
WIL has been abbreviated in comment.